### PR TITLE
Include robot assembly

### LINKED ISSUE #237: Include assembly process
## Desired behavior
It should be good to have in the `README.md` of the `andino_hardware` package the whole assembly process of the parts.

## Implementation suggestion
Using photos of the Ekuthon 2023, where 4 andinos were assembled, and adding photos of the Andino I have in Barcelona, we can create the assembly process in the `README.md` to help third party members build their own Andino.

### PR DESCRIPTION
**Pull request by JesusSilvaUtrera**
# 🎉 New feature

Closes #237 

## Summary
I have added to the `README.md` of the `andino_hardware` package the assembly steps.

## Test it
Just check the `README.md`

## Checklist
- [ ] Signed all commits for DCO
- [ ] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)

**Note to maintainers**: I will open the PR as a draft first to discuss what things to add, or whether the format is okay or not.

### CODE REVIEW

File: andino_hardware/README.md [hunk 1]
```diff
@@ -4,22 +4,71 @@ This package aims to provide the necessary information to the correct assembly o
...
 | Camera | [Raspi Camera Module V2, 8 MP](https://www.robotshop.com/products/raspberry-pi-camera-module-v2) | - | - |
 | Powerbank 5V | - | - | Any powerbank is suitable: Mind size / weight / output current(>=2A) |
 | (Optional) Power Step up | [DC - DC boost converter](https://www.amazon.com/0-9-Step-Regulator-DC-Converter/dp/B0C6QTJMFN/ref=sr_1_25?crid=G0FHM4SS5TWX&keywords=dc+step+up+converter&qid=1685741155&sprefix=dc+step+up+conver%2Caps%2C371&sr=8-25) | - | If motors support higher voltage than 5V a step-up(e.g: to 9V) can be added between powerbank(5V) and motor driver |
 | Fixing & Mount | M3 bolts/fasteners - M3 Spacers - M2.5/2.0 bolts/fasteners for SBC | - | - |
 
+## Assembly Process
+
+Following you can see a photo with all the parts:
+
+<img src="docs/Parts.jpg">
```
Comment (francocipollone) [inline andino_hardware/README.md:26]: You can add this as first step: 1. Gather all parts together
Add also a note here stating that the printed encoder wheels are not necessary.

File: andino_hardware/README.md [hunk 2]
```diff
@@ -4,22 +4,71 @@ This package aims to provide the necessary information to the correct assembly o
...
+
+The steps to reproduce the mounting of the Andino are:
+
+  1. Screw the Motor Driver and the idler wheel to the lower chassis.
+
+  <img src="docs/Caster_wheel_H_bridge.jpg">
```
Comment (francocipollone) [inline andino_hardware/README.md:32]: let's standarize the name of the files to be all snake_case

File: andino_hardware/README.md [hunk 3]
```diff
@@ -4,22 +4,71 @@ This package aims to provide the necessary information to the correct assembly o
...
+
+  Here you have a front image of how the idler wheel are mounted to the chassis:
```
Comment (francocipollone) [inline andino_hardware/README.md:34]: let's use caster instead of idler

**Review by francocipollone — CHANGES_REQUESTED** [review]
Looking good! Just dropped some comments :rocket:

File: andino_hardware/README.md [hunk 4]
```diff
@@ -21,36 +19,37 @@ This package aims to provide the necessary information to the correct assembly o
 
 ## Assembly Process
 
-Following you can see a photo with all the parts:
+The steps to reproduce the mounting of the Andino are:
```
Comment (francocipollone) [inline andino_hardware/README.md:22]: Being super obnoxious (nit ^10) :nerd_face:  : I guess we can remove this line as the section and steps are self explanatory

**Review by francocipollone — APPROVED** [review]

File: andino_hardware/docs/parts.png
Comment (francocipollone) [inline andino_hardware/docs/parts.png:1]: Wow 26 mb.

Comment (francocipollone) [reply andino_hardware/docs/parts.png:1]: Btw I love the numbers for guiding.

**Review by francocipollone — COMMENTED** [review]
Just noticed the following: the size of the images.

In order to avoid having a heavyweight repo, could you try to compress the images even more? I think that having images up to 1mb would be enough. (in case we notice the quality is too bad we could be stretch up to 2mb).

File: andino_hardware/docs/parts.png
Comment (JesusSilvaUtrera) [reply andino_hardware/docs/parts.png:1]: Yeah I thought about it this morning too, I will do that tomorrow, let's see if the image with the numbers can be compressed uo to 2 mb and look good, with the others there shouldn't be any problem.

Comment (JesusSilvaUtrera) [reply andino_hardware/docs/parts.png:1]: By the way, do you know which tool was used for the connections diagram? I was thinking of using GIMP to create it but just for curiosity.

Comment (francocipollone) [reply andino_hardware/docs/parts.png:1]: Super handmade. Check this: https://docs.google.com/drawings/d/1wummJPj3EJHSpB-iakCe97DVVQHOPI6oLSaNHQ7MesI/edit

Comment (JesusSilvaUtrera) [reply andino_hardware/docs/parts.png:1]: Great, thanks!

**Comment by francocipollone** [comment]
Just passing by, I haven't reviewed as it is a draft, let me know otherwise :+1:

**Comment by JesusSilvaUtrera** [comment]
@francocipollone @jballoffet this PR is ready for review, I have added the photo of the completed assembly. I know there are some problems with DCO from long ago and I noticed now, but I tried to do the rebase and they are a lot of commits to solve conflicts, so in the meantime you can take a look at the PR to give feedback if there is anything else you'd like to introduce, so we have time before RosCon.

Thanks in advance to both!

**Comment by JesusSilvaUtrera** [comment]
closing as there are too many conflicts to rebase, I will open a new fresh PR with all the final changes, so it's easier to review.

**Pull request closed without merging**